src/modules/tasksApp.js

```diff
@@ -51,12 +53,9 @@ const tasksApp = () => {
 
   // Retrieve the listArray from Local Storage when the page is loaded..............
 
```

Comment:
- [ ] After you made the request changes stated by the previous reviewer, there is another bug which after adding new todos, appears as undefined but it is saved in local storage. Also, I can't edit the todo, kindly fix it also.

Reply:
Please Note that you can't edit until you select the task in the three dots listed on the left this is one of the requirements in the Big picture about the project mentioned in the following video: 
[Big picture about the project ](https://www.youtube.com/watch?v=AcUd-_Yjjqg&ab_channel=EmiliaAndrzejewska)
Note : please for the next reviewer watch this video before reviewing